### PR TITLE
[2.3] Add check for app creation

### PR DESCRIPTION
**Pull request by mrajashree**
Add check for app creation

### CODE REVIEW

File: tests/integration/suite/test_project_catalog.py [hunk 1]
```diff
@@ -211,6 +212,59 @@ def test_user_addition_after_creating_project_catalog(admin_mc,
...
+
+    name = random_str()
+    new_project = client.create_project(name=random_str(), clusterId='local')
+    remove_resource(new_project)
+    wait_for_condition('InitialRolesPopulated', 'True', client, new_project)
+    new_project = client.reload(new_project)
+
+    project_name = str.lstrip(new_project.id, "local:")
+    catalog_name = project_name + ":" + name
+    url = "https://github.com/mrajashree/charts.git"
```
Comment (prachidamle) [inline tests/integration/suite/test_project_catalog.py]: Do we have a QA or test charts repo - we should have one

Comment (mrajashree) [reply tests/integration/suite/test_project_catalog.py]: no we don't, but yeah we should

Comment (prachidamle) [reply tests/integration/suite/test_project_catalog.py]: or we can use rancher charts repo itself

Comment (mrajashree) [reply tests/integration/suite/test_project_catalog.py]: few tests were using rancher charts repo, but there was once a change in rancher charts that removed a version and all these tests started failing.

Comment (dramich) [reply tests/integration/suite/test_project_catalog.py]: We do have one `DEFAULT_CATALOG = "https://github.com/rancher/integration-test-charts"`

Comment (mrajashree) [reply tests/integration/suite/test_project_catalog.py]: okay, I wasn't aware of this and have been using my fork for charts in tests. updated it

**Comment by prachidamle** [comment]
LGTM

**Comment by alena1108** [comment]
@mrajashree what issue does it address? It's better to have a github ticket with the clear steps to reproduce for qa

**Pull request merged**